metrics/prometheus.go

```diff
@@ -493,6 +493,11 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 		}
 		baseLabelValues := []string{id, name, image}[:len(baseLabels)]
 
+		for labelKey, labelValue := range container.Spec.Labels {
+			baseLabels = append(baseLabels, labelKey)
```

Comment:
`labelKey` will need to be sanitized as Prometheus labels need to match [this regexp](https://github.com/prometheus/client_golang/blob/8c899cd8208a366b6899d397e4bc3ced7ab073e3/prometheus/desc.go#L19).

You can copy the sanitize method from [here](https://github.com/prometheus/prometheus/blob/b964da4b75a589a6ded82becf9c456d1ac79b25d/util/strutil/strconv.go#L110-L114) & please add a test for this.